### PR TITLE
changelog: docs: API v1 deprecation note

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issues: https://tracker.mender.io/browse/MEN-1987

### CODE REVIEW

File: docs/management_api.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ info:
   title: Device authentication
   description: |
       An API for device authentication handling.
+      This API version has been deprecated and will be removed in Mender 1.8 release.
```
Comment (maciejmrowiec) [inline docs/management_api.yml]: Admission needs to get also a note

Comment (kjaskiewiczz) [reply docs/management_api.yml]: yes; I need to add a note to 1.6 admission api and remove admission api from master/1.7

Comment (kjaskiewiczz) [reply docs/management_api.yml]: 1.6 release contains admission 1.4, so is it ok to just add a note to admission 1.4? or do I need to create separate deviceadm branch - 1.6.x, add a note there and update integration 1.6 to contain admission 1.6?

**Comment by kacf** [comment]
I'm not sure why the license test started failing now, that requirement has been there for quite a while already. But the solution is easy, just update the copyright year in all the files it failed on.

File: docs/management_api.yml (hunk 1 above)
Comment (kacf) [reply docs/management_api.yml]: I think you mean [this one](https://github.com/mendersoftware/deviceadm/commit/54185dae47189d7db3c4bdd07bce8cb6f391de5a), right? I submitted it yesterday.

**Pull request merged**